doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -73,50 +73,49 @@ sdk\<service name>\<package name>\<package name>.sln
 
   Make sure `npx cadl compile .` runs correctly.
 
-  ***Add cadl-csharp emitter***
-
-  Modify package.json, add one line under dependencies:
-
-```diff
-        "dependencies": {
-          "@cadl-lang/compiler": "^0.37.0",
-          "@cadl-lang/rest": "^0.19.0",
-          "@azure-tools/cadl-azure-core": "^0.9.0",
-+         "@azure-tools/cadl-csharp": "0.1.8"
-        },
-```
-
-        Run `npm install` again to install @azure-tools/cadl-csharp.
-
-  **Notes**: @azure-tools/cadl-csharp: "0.1.8" only works with @cadl-lang/compiler: "0.37.0" and @cadl-lang/rest: "0.19.0"
-  
-
-  ***Modify (or create) cadl-project.yaml, add one line under emitters:***
-
-  ```diff
-  emitters:
-+  "@azure-tools/cadl-csharp": true
-  ```
+  **Note**: It is recommended to configure Cadl Specs on [REST API specifications](https://github.com/Azure/azure-rest-api-specs). Please refer to the [Guidelines](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/cadl-structure-guidelines.md).
 
   ***Configuration (optional)***
 
   One can further configure the SDK generated, using the emitter options on @azure-tools/cadl-csharp.
 
-```
-emitters:
+  Modify (or create) cadl-project.yaml, add emitter options under options/@azure-tools/cadl-csharp.
+
+```diff
+parameters:
+  "python-sdk-folder":
+    default: "{cwd}/azure-sdk-for-python/"
+  "java-sdk-folder":
+    default: "{cwd}/azure-sdk-for-java/"
+  "js-sdk-folder":
+    default: "{cwd}/azure-sdk-for-js/"
+  "chsarp-sdk-folder":
+    default: "{cwd}/azure-sdk-for-chsarp/"
+  "openapi-folder":
+    default: "{project-root}/../dataplane/"
+  "service-directory-name":
+    default: mycadlproject
+emit:
+  - "@azure-tools/cadl-autorest"
+options:
   "@azure-tools/cadl-csharp":
-    namespace: "Azure.IoT.DeviceUpdate"
-    clear-output-folder: true
++    namespace: Azure.Template.MyCadlProject
++    "emitter-output-dir": "{csharp-sdk-folder}/sdk/{service-directory-name}/{namespace}/src"
 ```
 
   **Notes**:
   
   @azure-tools/cadl-csharp emitter options:  
 
 - `namespace` define the client library namespace. e.g. Azure.IoT.DeviceUpdate.
-- `new-project` indicate if it is a new sdk project and need to generate a project file (.csproj).
-- `model-namespace` Indicate if we want to put the models in their own namespace which is a sub namespace of the client library namespace plus ".Models". if it is set `false`, the models will be put in the same namespace of the client. The default value is `true`.
+- `emitter-output-dir` define the output dire path which will store the generated code.
+- `generate-protocol-methods` indicate if you want to generate **protocol method** for each operation. By default is true.
```

Comment:
This needs to change to the [latest global names](https://github.com/Azure/cadl-azure/pull/2527/files#diff-056dab893146ee549ee9d568342b425963fd2b2561b76b7a4c31ab2b05f75da6R75-R76) or be omitted so it doesn't cause confusion.

Reply:
`generate-protocol-method` and `generate-convenience-method` those two options defined as emitter options if customer want to change the value.